### ISSUE TITLE
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in wp-includes\post-template.php on line 293

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#2549]
https://pastebin.com/fEg8pu5R

**Comment by EkoJR** [comment]
This was introduced with https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/2410

This occurs with all WP versions below 5.2. Updating WP should fix the problem; for those who can update.

When breakpointing, I noticed WP doesn't use a WP_Post param, but uses the global `$post` instead, and on a sitemap page the global `$post` is null.

**Comment by michaeltorbert** [comment]
@EkoJR Can it be fixed for 4.6<x<5.2?

**Comment by EkoJR** [comment]
Yes.

```
get_the_excerpt( $post->ID )
```

change to
```
get_post_field( 'post_excerpt', $post->ID )
```
